Jobs.Fetcher.TikTok/Helpers/ApiDataFetcher.cs

```diff
@@ -0,0 +1,242 @@
+                                                                    AvatarThumbnail = authorJSON["avatarThumb"].ToString(),
+                                                                    AvatarMedium = authorJSON["avatarMedium"].ToString(),
+                                                                    AvatarLarge = authorJSON["avatarLarger"].ToString(),
+                                                                    Signature = authorJSON["signature"].ToString(),
+                                                                    Verified = authorJSON["verified"].ToObject<bool>(),
+                                                                    SecurityUId = authorJSON["secUid"].ToString(),
+                                                                    Secret = authorJSON["secret"].ToObject<bool>(),
+                                                                    FTC = authorJSON["ftc"].ToObject<bool>(),
+                                                                    Relation = authorJSON["relation"].ToObject<int>(),
+                                                                    OpenFavorite = authorJSON["openFavorite"].ToObject<int>(),
+                                                                    CommentSetting = authorJSON["commentSetting"].ToObject<int>(),
+                                                                    DuetSetting = authorJSON["duetSetting"].ToObject<int>(),
+                                                                    StitchSetting = authorJSON["stitchSetting"].ToObject<int>(),
+                                                                    PrivateAccount = authorJSON["privateAccount"].ToObject<bool>()*/
+            };
+        }
+
+        public static AuthorStats GetTikTokAuthorStatsFromJSON(JToken authorStatsJSON, Author author, DateTime postTime) {
+            return new AuthorStats() {
+                       FollowingCount = authorStatsJSON["followingCount"].ToObject<long>(),
+                       FollowerCount = authorStatsJSON["followerCount"].ToObject<long>(),
+                       HeartCount = authorStatsJSON["heartCount"].ToObject<long>(),
+                       VideoCount = authorStatsJSON["videoCount"].ToObject<long>(),
+                       DiggCount = authorStatsJSON["diggCount"].ToObject<long>(),
+                       Heart = authorStatsJSON["heart"].ToObject<long>(),
+                       EventDate = DateTime.Today,
+                       ValidityStart = DateTime.UtcNow,
+                       ValidityEnd = DateTime.MaxValue,
+                       AuthorId = author.Id,
+                       Author = author
+            };
+        }
+
+        public static Post GetTikTokPostFromJSON(JToken postJSON, Author author, Video video, Music music, List<string> challengeIds, List<string> tagIds, List<string> effectStickerIds) {
+            return new Post() {
+                       Id = postJSON["id"].ToString(),
```

Comment:
I don't see any error handling. What would happen in case the JSON payload does not contain the fields we were expecting?